### ISSUE TITLE
[Bug] Teleporting Vehicles & Players after being created

### ISSUE THREAD

**Issue by parpMoretti** [altmp/altv#152]
When creating a vehicle in c# serverside the vehicle position doesn't work correctly.

When I spawn the vehicle, it'll spawn at the position. If I teleport the vehicle it doesn't work.

As an example:

I do a command called /vget, we then spawn the vehicle and teleport it to the player. The vehicle spawns and position gets set but the vehicle doesn't seem to physically move. The vehicle is out of streaming range.

```
VDimension: 0, ID: 1, Position: Position(x: -1180.09, y: -1486.16, z: 4.28776)
Vehicle Pos: Position(x: 200.6505, y: -932.0967, z: 30.67834) to your position: Position(x: 200.6505, y: -932.0967, z: 30.67834) - Distance: 0
```

Here, I spawn the vehicle then teleport to the player but the vehicle doesn't appear.

If I spawn the vehicle at the player position, it works and appears.
```
VDimension: 0, ID: 1, Position: Position(x: 200.6505, y: -932.0967, z: 30.67834)
Vehicle Pos: Position(x: 200.6505, y: -932.0967, z: 30.67834) to your position: Position(x: 200.6505, y: -932.0967, z: 30.67834) - Distance: 0
```

**Comment by FabianTerhorst** [comment]
tl;dr position setter doesn't physically moves vehicle but changes internal data.

**Comment by vadzz-dev** [comment]
Duplicate of #71
